server/models/User.js

```diff
@@ -24,5 +24,42 @@ const UserSchema = new Schema({
 	}
 });
 
+// Add a new user
+UserSchema.statics.addNewUser = function(firstName, lastName, email, password){
+	return this.insert({
+		"firstName": firstName,
+		"lastName": lastName,
+		"email": email,
+		"password":password	
```

Comment:
We're going to want to hash the password, you can take a look at bcrypt for hashing and comparing